its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -191,12 +192,20 @@ public static Integer getMeasureAsInt(Orchestrator orch, String componentKey, St
   }
 
   @CheckForNull
-  public static org.sonarqube.ws.Duplications.ShowResponse getDuplication(Orchestrator orch, String componentKey) {
-    org.sonarqube.ws.Duplications.ShowResponse response = newWsClient(orch).duplications().show(new org.sonarqube.ws.client.duplications.ShowRequest()
+  public static ShowResponse getDuplication(Orchestrator orch, String componentKey) {
+    ShowResponse response = newWsClient(orch).duplications().show(new org.sonarqube.ws.client.duplications.ShowRequest()
```

Comment:
I think we can remove `org.sonarqube.ws.client.duplications.` too.

Reply:
ShowRequest class is already imported from package: `org.sonarqube.ws.client.components` which means we have to use the fully qualified name here, otherwise there will be a clash.